### ISSUE TITLE
Windows Security prompts even though no passkey

### ISSUE THREAD

**Issue by dequeues** [smartfrigde/armcord#726]
**Describe the bug**
Whenever I open ArmCord, it prompts me to sign in with a security key. I *did* have a key previously, but I have removed this. I have tried to uninstall and reinstall ArmCord, but the issue persists.

https://i.imgur.com/4KRYDFg.png

This *only* happens with ArmCord. I can open Discord with no popup.

**To Reproduce**

**Expected behavior**
It shouldn't prompt me to sign in to discord.com

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows
 - Method of installation setup
 - Version [e.g. 3.0.7] 3.3.1

**Additional context**

**Comment by SomeAspy** [comment]
does this happen if you remove the account from the client and try to login again?
I assume this has to do with some sort of caching discord is doing, because we don't touch the login flow.

**Comment by dequeues** [comment]
> does this happen if you remove the account from the client and try to login again? I assume this has to do with some sort of caching discord is doing, because we don't touch the login flow.

yeah, I've tried to login and logout and to even try to recreate a security key and then remove it, but still the same. It is interesting how it is tied to ArmCord though, I've tried to rename the process, so I assume it's relating to some CLSID I'm trying to dig through with procmon

**Comment by dequeues** [comment]
Been looking for past little while and I've not gotten any closer - I don't know where this would hide in the registry, nothing returns with `discord.com` when searching, neither.

**Comment by dequeues** [comment]
When I set up the same with DiscordPtb, it doesn't automatically pop up unless I press the "Sign in with passkey" under the QR code.

**Comment by smartfrigde** [comment]
I've been getting these recently on fresh installs, on web too. I'm pretty sure this is a Discord bug. Nothing in ArmCord should trigger this

**Comment by dequeues** [comment]
Discord told me to contact Microsoft. This shall be a fun ride.

**Comment by dequeues** [comment]
Something I did notice is that removing a passkey with LegCord leaves a remnant in Accounts > Passkeys

**Comment by dequeues** [comment]
Seems like Discord does this, too. This is frustrating because it only happens with Legcord, even after being renamed, yet no such pop-up happens with the official Discord client. Any suggestions? @smartfrigde @SomeAspy

**Comment by dequeues** [comment]
Looked into this some more. This is the culprit line:
```javascript
null == m.Z && null != window.PublicKeyCredential && null != PublicKeyCredential.isConditionalMediationAvailable && PublicKeyCredential.isConditionalMediationAvailable().then(e => {
```

The difference is, m.Z on normal Discord has an object:
```javascript
{
    "Z": {
        "isRenderer": true,
        "nativeModules": {
            "canBootstrapNewUpdater": false
        },
        "process": {
            "pid": 27844,
            "platform": "win32",
            "arch": "x64",
            "env": {
                "LOCALAPPDATA": "C:\\Users\\natha\\AppData\\Local",
                "PROGRAMFILES(X86)": "C:\\Program Files (x86)",
                "PROGRAMFILES": "C:\\Program Files",
                "PROGRAMW6432": "C:\\Program Files",
                "PROGRAMDATA": "C:\\ProgramData"
            }
        },
        "os": {
            "appArch": "x64",
            "arch": "x64",
            "release": "10.0.26100"
        },
        "app": {
            "dock": {}
        },
        "clipboard": {},
        "ipc": {},
        "gpuSettings": {},
        "window": {
            "USE_OSX_NATIVE_TRAFFIC_LIGHTS": true
        },
        "powerMonitor": {},
        "spellCheck": {},
        "crashReporter": {},
        "desktopCapture": {},
        "fileManager": {},
        "clips": {},
        "processUtils": {},
        "powerSaveBlocker": {},
        "http": {},
        "accessibility": {},
        "features": {},
        "settings": {},
        "userDataCache": {},
        "thumbar": {},
        "safeStorage": {},
        "globalOverlay": {
            "WINDOW_KEY": "DISCORD_OutOfProcessOverlay"
        },
        "hardware": {},
        "riotGames": {},
        "remoteApp": {
            "dock": {}
        },
        "remotePowerMonitor": {},
        "webAuthn": {}
    }
}
```

whereas, on Legcord, m.Z is undefined. Could it have something to do with the tracking being blocked? @smartfrigde

**Comment by JustASpeedrunner** [comment]
Hi, I am also having this issue (first time install). Any updates/workarounds that haven't been posted here?

**Comment by dequeues** [comment]
@smartfrigde did you manage to look into this?

**Comment by GriffinCodes** [comment]
I am also running into this issue, I cannot get Legcord to open because of it. v1.1.1 on win11 arm64

**Comment by JasonCraneAdkins** [comment]
Same here,

**Comment by anon88391** [comment]
I'm also experiencing the same issue since a recent Legcord update, it's very annoying to have to type in my password every time I open the app.
This started happening out of nowhere and I can't recall ever setting up a passkey on my account.

**Comment by I2pRandom** [comment]
Encountering the same problem, even when utilizing group policy keys to disable the pass-key authentication behavior on Window's it's self. 

<img width="1283" height="524" alt="Image" src="https://github.com/user-attachments/assets/44405b22-233f-45ad-8c7a-292dab8e47d3" />

**Comment by MrAydinminer** [comment]
I'm having the same issue

**Comment by anon88391** [comment]
Still experiencing this issue as of the latest 1.1.6 update.
It doesn't happen if I remember to manually close Legcord before shutting my PC off, but if I forget then on the next start I will be logged out and it will throw the passkey prompt at me.
It's driving me insane to have to log in after every reboot and I'll have to stop using Legcord until it's fixed. 
I don't understand why it's not happening to everyone, I've tried so many things to fix it but it keeps happening.
Is there any chance the fixing priority on this could be increased? @smartfrigde

**Comment by SimonYuuki** [comment]
I also have this prompt with 1.2.1 on Windows... No idea why is it doing that

**Comment by dequeues** [comment]
If possible, could some people test #1033? It seems to work for me. No prompt for no passkey, and shows a prompt when clicking sign in with passkey.

**Comment by dequeues** [comment]
@SimonYuuki do you still get this issue? If so, can you try this?

On another note, I've been using this patch and I've had no issues. I sign in with a security key, which you click sign in with security key and prompt shows as it should.
